src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.cs

```diff
@@ -78,7 +78,14 @@ public void RaByResource()
         {
             ResourcesController.NewInstance.RunPsTest("Test-RaByResource");
         }
-        
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void RaValidateInputParameters()
```

Comment:
I think this is necessary, but not sufficient.  Please include some unit tests of just the validation method.  This will allow you to easily cover all that code without requiring a powershell invocation.  Make sure to cover each of the negative and positive outcomes [/, tenant resources, subscription, resource group, top-level resource, child resource, grandchild resource]